.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
This is a bit unrelated to this PR and not sure it belongs in this project, I'd say this is something you could add to your global .gitignore:
https://stackoverflow.com/a/7335487

```suggestion
```

Reply:
removed.